examples/webgl_instancing.html

```diff
@@ -0,0 +1,129 @@
+<!DOCTYPE html>
+<html lang="en">
+	<head>
+		<title>three.js webgl - instancing</title>
+		<meta charset="utf-8">
+		<meta name="viewport" content="width=device-width, user-scalable=no, minimum-scale=1.0, maximum-scale=1.0">
+		<link type="text/css" rel="stylesheet" href="main.css">
+	</head>
+	<body>
+
+		<script type="module">
+
+			import * as THREE from '../build/three.module.js';
+
+			import Stats from './jsm/libs/stats.module.js';
+
+			var camera, scene, renderer, stats;
+
+			var amount = 12;
+			var count = Math.pow( amount, 3 );
+
+			init();
+			animate();
+
+			function init() {
+
+				camera = new THREE.PerspectiveCamera( 60, window.innerWidth / window.innerHeight, 0.1, 1000 );
+				camera.position.set( 10, 10, 10 );
+				camera.lookAt( 0, 0, 0 );
+
+				scene = new THREE.Scene();
+
+				var geometry = new THREE.BoxBufferGeometry( 1, 1, 1 );
+				var material = new THREE.MeshNormalMaterial();
+				// var material = new THREE.MeshBasicMaterial( { color: 0xff0000, opacity: 0.1, transparent: true } ); // overdraw
+
+			 	var mesh = new THREE.InstancedMesh( geometry, material, count );
+				scene.add( mesh );
+
+				var loader = new THREE.BufferGeometryLoader();
+				loader.load( 'models/json/suzanne_buffergeometry.json', function ( geometry ) {
+
+					geometry.computeVertexNormals();
+					geometry.scale( 0.5, 0.5, 0.5 );
+
+					mesh.geometry = geometry;
+
+				} );
+
```

Comment:
You should have only one geometry pushed to the GPU.

This works.

```js
var material = new THREE.MeshNormalMaterial();

new THREE.BufferGeometryLoader().load( 'models/json/suzanne_buffergeometry.json', function ( geometry ) {

	geometry.computeVertexNormals();
	geometry.scale( 0.5, 0.5, 0.5 );

	var mesh = new THREE.InstancedMesh( geometry, material, count );
	scene.add( mesh );

} );
```

Reply:
The example is still work in progress but... it actually shows that you can easily change geometries at any point, which is something that other PRs didn't make it easy.

Reply:
Then, I guess you should dispose of the old geometry...

Reply:
Actually, users copy these examples, so I still would suggest using the conventional pattern I posted above, and then add a GUI that allows the user to switch the geometry and/or material (if you feel it is important to demonstrate that capability). JMO.